src/app/pages/public/CreateLifePage/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Form, Input, Button, DatePicker } from 'antd';
+import { useNavigate } from 'react-router-dom';
+import BasicLayout from '../../../layouts/BasicLayout';
+import { useCreateLifeMutation } from '../../../api';
+
+const validateMessages = {
+    required: '${label} is required!',
+};
+
+const layout = {
+    labelCol: {
+        span: 8,
+    },
+    wrapperCol: {
+        span: 16,
+    },
+};
+
+const CreateLifePage = () => {
+    const navigate = useNavigate();
+
+    const { t } = useTranslation(['life'])
+
+    const [createLife] = useCreateLifeMutation();
+
+    const onFinish = useCallback(async values => {
+        const formData = {
+            ...values,
+        };
+
+        if (values['birthday']) {
+            formData['birthday'] = new Date(values['birthday']);
+        }
+
+        if (values['hobbies']) {
+            formData['hobbies'] = values['hobbies'].split('\n');
+        }
+
+        await createLife({ variables: { life: formData } });
+
+        navigate('/life');
+    }, []);
+
+    return (
+        <BasicLayout>
+            <Form {...layout} name="nest-messages" onFinish={onFinish} validateMessages={validateMessages}>
+                <Form.Item
+                    name={'firstname'}
```

Comment:
If it's only a string, you don't need to wrap this in JS expression `{}` and can directly use it as a string `name="firstname"`